projects/kubernetes/cloud-provider-vsphere/1-26/GIT_TAG

```diff
@@ -0,0 +1 @@
+v1.25.0
```

Comment:
This is going to mean that the 1.25 build and the 1.26 are going to push to the same image tag. Its going to be the same content so I cant think of a reason it will matter but just calling it out. cc @abhay-krishna